docs/deprecated.md

```diff
@@ -84,10 +84,10 @@ Because of which, the driver specific log tag options `syslog-tag`, `gelf-tag` a
 ### LXC built-in exec driver
 **Deprecated In Release: v1.8**
 
-**Target For Removal In Release: v1.10**
+**Removed In Release: v1.10**
 
 The built-in LXC execution driver is deprecated for an external implementation.
-The lxc-conf flag and API fields will also be removed.
+The lxc-conf flag and API fields are also removed.
```

Comment:
Sorry my bad. Forgot the Ln 89. Updated the PR again. Thanks for the help @thaJeztah.